blocks/basic/include/gnuradio-4.0/basic/ConverterBlocks.hpp

```diff
@@ -0,0 +1,267 @@
+#ifndef CONVERTERBLOCKS_HPP
+#define CONVERTERBLOCKS_HPP
+
+#include <tuple>
+
+#include <gnuradio-4.0/Block.hpp>
+#include <gnuradio-4.0/BlockRegistry.hpp>
+#include <gnuradio-4.0/DataSet.hpp>
+#include <gnuradio-4.0/meta/UncertainValue.hpp>
+
+namespace gr::blocks::type::converter {
+
+template<typename T, typename R>
+requires std::is_arithmetic_v<T> && std::is_arithmetic_v<R>
+struct Convert : public gr::Block<Convert<T, R>> {
+    using Description = Doc<"(@brief basic block to perform a input to output data type conversion (N.B. w/o scaling))">;
+    PortIn<T>  in;
+    PortOut<R> out;
+
+    template<gr::meta::t_or_simd<T> V>
+    [[nodiscard]] constexpr auto processOne(const V& input) const noexcept {
+        if constexpr (gr::meta::any_simd<V>) { // simd case
+            using RetType = std::experimental::rebind_simd_t<R, V>;
```

Comment:
```suggestion
            using RetType = vir::stdx::rebind_simd_t<R, V>;
```
clang-18 does not implement it yet.